client/src/components/SeasonCollectionImageCardComponent/SeasonCollectionImage.css

```diff
@@ -0,0 +1,32 @@
+.season-collection-text {
+  font-family: "Playfair Display", serif;
+  text-align: center;
+  font-size: 14px;
+  padding-top: 0px;
+  padding-bottom: 5px;
+  /* figma shows bold but bold does not work */
+  font-weight: bold;
+  margin: 5px;
+}
+.season-collection-image-container {
+  display: flex;
+  margin: 0px;
+  padding: 0px;
+  justify-content: center;
+}
```

Comment:
I think you may need to set a max width or something here. It looks good, but it starts to get really big when you expand the display. Can you work on making this more responsive?